sdk/src/main/scala/com/horizen/forge/ForgeMessageBuilder.scala

```diff
@@ -0,0 +1,133 @@
+package com.horizen.forge
+
+import com.horizen.block.{MainchainBlockReference, SidechainBlock}
+import com.horizen.box.{ForgerBox, NoncedBox}
+import com.horizen.companion.SidechainTransactionsCompanion
+import com.horizen.consensus._
+import com.horizen.params.NetworkParams
+import com.horizen.proof.VrfProof
+import com.horizen.proposition.Proposition
+import com.horizen.secret.{PrivateKey25519, VrfSecretKey}
+import com.horizen.transaction.SidechainTransaction
+import com.horizen.utils.MerklePath
+import com.horizen.{SidechainHistory, SidechainMemoryPool, SidechainState, SidechainWallet}
+import scorex.core.NodeViewHolder.ReceivableMessages.GetDataFromCurrentView
+import scorex.util.{ModifierId, ScorexLogging}
+
+import scala.util.{Failure, Success}
+
+case class ForgerDataWithSecrets(forgerBox: ForgerBox, merklePath: MerklePath, forgerBoxRewardPrivateKey: PrivateKey25519, vrfSecret: VrfSecretKey)
```

Comment:
not used anymore. Can be removed or modified and used as a result of `getSecretsAndProof`

Reply:
removed